static/coffee/controllers.coffee

```diff
@@ -573,12 +588,12 @@ controllers.controller('CaseController', ['$scope', '$window', '$timeout', 'Case
   #----------------------------------------------------------------------------
 
   $scope.onWatch = () ->
-    UtilsService.confirmModal("Receive notifications for activity in this case?").then(() ->
+    UtilsService.confirmModal("Receive alerts for activity in this case?").then(() ->
```

Comment:
/me nods

I'll give replacing them a try in this PR to get a sense of the level of effort required, will see from there.